src/ConvertToSarif/ConvertOptions.cs

```diff
@@ -3,28 +3,20 @@
 
 using CommandLine;
 
-namespace Microsoft.CodeAnalysis.Sarif.ConvertToSarif
```

Comment:
>ConvertToSarif [](start = 39, length = 14)

I don't agree with pushing the classes in this tool up into the base SARIF namespace.